wiki/Beatmap_Editor/Song_Setup/en.md

```diff
@@ -1,222 +1,239 @@
-    -   Note: If you have a tag of "qwertyuiop" as a tag on your beatmap, you can also find your beatmap in-game by typing in "qwertyuiop".
+## Difficulty
 
-Again, to avoid issues, please make sure that everything (aside from Difficulty name) is filled out identically for each difficulty.
+![Difficulty section. The right section is for osu!mania-specific beatmap.](img/SS_Difficulty.jpg "Difficulty section. The right section is for osu!mania-specific beatmap.")
 
-Difficulty
-----------
+Beatmap's metrics settings. Each setting affects different aspects of the beatmap.
 
-![Difficulty section](img/SS_Difficulty_Settings.jpg "Difficulty section")
-
-Tab related to the core of beatmap's settings. Each setting affects different aspects of the beatmap.
+Holding down `Shift` button allows for a precise 0.1 value adjustment over the standard 1 value adjustment.
 
 ### HP Drain Rate
 
-Known as "**HP**" in Edit's song selection.
+HP Drain Rate, also known as HP, determines the rate of health recovery from good scores and health punishment from bad scores. The value ranges from 0 to 10; higher HP value leads to lower health recovery and higher health punishment.
 
-The rate at which HP decreases, which is always constant except during [breaks](/wiki/Glossary). Further HP is lost with misses. HP is recovered by getting a [score value](/wiki/Score) and spinning spinners. More HP is recovered with [Geki and Katu](/wiki/Score) at the end of a combo.
+In osu!standard and osu!catch, it is also the rate at which HP decreases (which is always constant except during the [breaks](/wiki/Breaks)). Note that more HP is recovered with [Geki and Katu](/wiki/Score/#osu) at the end of a combo in osu!standard.
 
-### Circle Size (osu!, osu!catch, osu!taiko only)
+### Circle Size
 
 ![Overview of various Circle Sizes](img/Circle_sizes.png "Overview of various Circle Sizes")
 
-Known as "**CS**" in Edit's song selection.
-
-This determines the size of circles and sliders. The value range from 2 to 7 with the smaller numbers being large sizes (like needle gauges). The smaller circles make the map harder by requiring the player to be more precise.
-
-Most maps use sizes 3, 4 and 5. 6 is sometimes used and 7 is almost never used (it will be roughly the size of your cursor). 7 is likely to be seen when using the [Hard Rock](/wiki/Game_Modifiers) mod. An example of [beatmap using 3, 5, 6, and 7 as default](https://osu.ppy.sh/s/2008) and [beatmap that uses 0, 1, 3 and 4 as default](https://osu.ppy.sh/s/1208).
+Circle Size, also known as CS, exists only in osu!standard mode and osu!catch mode. CS determines the size of hit objects. However, in osu!catch, it also determines the catcher's size. The values range from 2 to 7; higher CS values will make hit objects smaller. The highest value will make them exactly the size of the cursor.
```

Comment:
Do you need the `mode` suffix after the game modes' names? Not consistent throughout game mode names in this article. (i.e. some have `mode`, some don't)